code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -160,7 +160,8 @@
 					if(!foundghost)
 						msg += " and [p_their()] soul has departed"
 			msg += "...</span>\n"
-
+	for(var/mob/living/simple_animal/hostile/poison/bees in stomach_contents) //Get bloated mouth from eating a bee
```

Comment:
Not sure if adding the text like this is the way to go, I can imagine someone eating a ton of bees and having an insanely long examine text. you might just want to check something like `is_type_in_list()`

Reply:
One would only be capable of having two examine texts at maximum, it doesnt stay for long and it takes a while to devour the bee.
Do you have any idea how i could do that? Been trying to use it with no success.

Reply:
I think you can use it like
```
if(is_type_in_list(/mob/living/simple_animal_hostile/poison/bees, stomach_contents))
    ...

```

Reply:
I tried that before, gives that error:
	"message": "/mob/living/simple_animal_hostile/poison/bees: undefined type path",
	"message": "failed to resolve path /mob/living/simple_animal_hostile/poison/bees",

Reply:
Oh, I put an extra underscore in there. Should be `simple_animal/hostile`. Maybe try again with the path fixed?

Reply:
Been stung so many times the brain is failing to process :)
But sadly it still doesnt work (no error), tested it before and now again.

Reply:
Removed examine text.